### ISSUE TITLE
Wayland LaunchBar not rendering image

### ISSUE THREAD

**Issue by 5ila5** [qtile/qtile#4552]
### Issue description

Using the LaunchBar widget configured like this: 
```python
widget.LaunchBar(progs=[("/home/silas/Nextcloud/uni/stundenplan.png", "mupdf /home/silas/Nextcloud/uni/stundenplan.png","stundenplan")]),
```
leads to not showing the image but a hole in my bar when using wayland (X11 works fine).

I tested with older versions of this widget. This widget works fine in Wayland before the changes in #3857 by @elParaguayo .

After revoking some changes by this pr it wokred again after replacing https://github.com/qtile/qtile/blob/1e0f09f45a4396d563039bf711ed82bdbeae8b36/libqtile/widget/launchbar.py#L291
with
```python
self.drawer.draw(offsetx=self.offset + xoffset, offsety=self.offsety, width=self.width)
```
Seems to work fine in X11 as well but I'm not sure if this may lead to other problems or if this is an appropriate fix for this bug as the X11 _version_ seems to render this differently than the wayland _version_

### Version

0.23.0

### Backend

Wayland (experimental)

### Required

- [X] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by elParaguayo** [comment]
Sorry for the delay.

They should render in exactly the same way so this change _shouldn't_ be necessary. I'll take a look.

**Comment by elParaguayo** [comment]
Please try #4592

**Comment by 5ila5** [comment]
Thank you.

Works again with this fix

**Comment by 5ila5** [comment]
You're right a smaller icon_size positions the icon at the top and is not centred (Wayland and X11)

**Comment by elParaguayo** [comment]
OK - can you try again. I think the resizing should work now.

**Comment by 5ila5** [comment]
looks fine. Icons are now vertically centred as well
